opencodelists/tests/test_import_coding_system_data.py

```diff
@@ -90,16 +90,16 @@ def test_calls_import_data_function_coding_system_force_overwrite(
 
 
 @pytest.mark.parametrize(
-    "input_date,error",
+    "input_date,error_regex",
```

Comment:
This is a really nice improvement. Knowing that the string is a regex immediately tells me why the parentheses are escaped.